sql/src/test/java/io/crate/integrationtests/UpdateIntegrationTest.java

```diff
@@ -381,17 +382,32 @@ public void testUpdateNestedObjectDeleteWithoutArgs() throws Exception {
     }
 
     @Test
-    public void testUpdateWithNestedObjectArrayIdxAccess() throws Exception {
+    public void testUpdateArrayElementIsNotAllowed() {
```

Comment:
And I'd move this one as a "Unit test" to UpdateAnalyzerTest. @seut what do you think?